OCAT/client/resources/js/components/Assessments/new.js

```diff
@@ -1,16 +1,102 @@
 import React from 'react';
+import { useForm } from "react-hook-form";
 import { AssessmentService } from '../shared/services/assessment.service';
 
 export function AssessmentNew(){
+  const createAssessmentObject = (data)=>{
+    var assessment ={
+      instrument: data.instruments, 
+      catName : data.catName,
+      catDateOfBirth : data.catDateOfBirth,
+    }
+    assessment.score = calculateScore(data)
+    assessment.riskLevel = calculateRiskLevel(assessment.score)
+    return assessment;
+  }
 
-    //create a form that utilizes the "onSubmit" function to send data to OCAT/client/libs and onto the OCAT/server/routes express API
-    const onSubmit = async (data) => {
-        await AssessmentService.submit(data);
-    };
+  const calculateScore = (data)=>{
+    var score = 0;
+    score += +data.prevJudSys;
+    score += +data.phsAltCat;
+    score += +data.prsAltOwn;
+    score += +data.plyDog;
+    score += +data.hssStr;
+    return score;
+  }
+
+  const calculateRiskLevel = (score)=>{
+    var riskLevel = 'low';
+    if(score > 1){
+      if(score <=3) riskLevel = 'meduim';
```

Comment:
```suggestion
      if(score <=3) riskLevel = 'medium';
```

Reply:
I understood that I just remove nested if statement.
So I changed this to be:
    let riskLevel = 'low';
    if(score > 1) riskLevel = 'meduim';
    if(score > 3) riskLevel = 'high';

Reply:
I was just correcting your spelling of the word medium.

But, in general, using a switch case is more efficient for things like this
https://www.w3schools.com/js/js_switch.asp

Reply:
I corrected the typo.